eval/evaluator.py

```diff
@@ -28,6 +34,47 @@ def __init__(self, config, analyzer=None, metric_handler=None, label_schema=None
         self.history = []
         self.analyzer = analyzer
         self.score_func = self.get_eval_function(config)
+        self.metric_dictionary = [
```

Comment:
This metric dictionary should be removed.
You are getting the dictionary from the previous line:
self.get_eval_function()

Reply:
Would you like me to move the dictionary into `get_eval_function`? Or remove it all together?

Reply:
Remove it all, since these are static metrics. We would like to not have static metrics, but all of them should be generated using in the `get_eval_function`, using the metric generator